### PR TITLE
Handle MemoryError and exit gracefully.

### PR DESCRIPTION
**Pull request by amjith**
## Description

Addresses #663 

## Checklist

- [x] I've added this contribution to the `changelog.md`.
- [x] I've added my name to the `AUTHORS` file (or it's already there).

### CODE REVIEW

File: mycli/main.py [hunk 1]
```diff
@@ -1159,7 +1159,13 @@ def cli(database, user, host, port, socket, password, dbname,
         mycli.run_cli()
     else:
         stdin = click.get_text_stream('stdin')
-        stdin_text = stdin.read()
+        try:
+            stdin_text = stdin.read()
+        except MemoryError:
+            click.secho('Failed! Ran out of memory.' ,err=True, fg='red')
+            click.secho('You might want to try the official mysql client.' ,err=True, fg='red')
+            click.secho('Sorry... :(' ,err=True, fg='red')
```
Comment (42B) [inline mycli/main.py]: Shouldn't the commas have no space preceding them but a single space afterwards too?

Comment (j-bennet) [reply mycli/main.py]: Looks like the lint checker actually complained about that: https://travis-ci.org/dbcli/mycli/jobs/456439367. And yet it exited with 0. We need to fix that.

Comment (amjith) [reply mycli/main.py]: Fixed the pep8 violation.

**Comment by j-bennet** [comment]
👍

**Pull request merged**